src/basic/log.c

```diff
@@ -405,7 +405,7 @@ static int write_to_console(
         if (show_location) {
                 const char *lon = "", *loff = "";
                 if (show_color) {
-                        lon = ANSI_HIGHLIGHT_YELLOW4;
+                        lon = ansi_highlight_yellow4();
```

Comment:
Is this correct? It is conditioned by `show_color` boolean, which may be different from SYSTEMD_COLORS.

Reply:
I'm not sure but I think it should always match `colors_enabled`, which is based on `get_color_mode`.
If I'm wrong, there could be a case with no colors even if `show_colors` is true, but that requires either `NO_COLOR` or `SYSTEMD_COLORS=0`, which makes sense to me.

Reply:
@yuwata this is the last outstanding comment - any update? Does the explanation make sense?